tests/pybaseball/analysis/projections/marcels/test_marcels_batting.py

```diff
@@ -1,5 +1,5 @@
-from pybaseball.analysis.projections.marcels import MarcelProjectionsBatting
-from pandas import DataFrame
+from pybaseball.analysis.projections.marcels import MarcelProjectionsBatting	
+from pandas import DataFrame	
```

Comment:
This file should not be in the PR at all, what happened here?

Reply:
Also you just fudged the whitespace. Why did you add these to the commit in the first place?

Reply:
Because you requested that I re-add the old testing, so these two files got caught up in the mix. Since I was just re-adding old files, I did not evaluate them individually.